WordPress/src/main/AndroidManifest.xml

```diff
@@ -220,7 +220,7 @@
             android:name=".ui.stories.StoryComposerActivity"
             android:label="@string/app_name"
             android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.Immersive">
+            android:theme="@style/WordPress.Stories.Immersive">
```

Comment:
this is a seemingly unrelated change - the action bar looked weird when triggering the media picker from the story composer, that's why it's here (we have had this theme already applied in the demo app correspondingly, although differently named)